package.json

```diff
@@ -43,9 +43,10 @@
         "sweetalert": "^2.1.2",
         "twitter-text": "^2.0.5",
         "vue-content-loader": "^0.2.1",
-        "vue-infinite-loading": "^2.4.4",
-        "vue-loading-overlay": "^3.2.0",
-        "vue-timeago": "^5.1.2"
+        "vue-infinite-loading": "^2.4.3",
```

Comment:
You based this PR on ```dev``` instead of ```frontend-ui-refactor``` which is okay, but we have updated npm packages since then so can you please use the latest versions:

```"vue-infinite-loading": "^2.4.4",```
```"vue-loading-overlay": "^3.2.0",```
```"vue-timeago": "^5.1.2"```